src/liballoc/rc.rs

```diff
@@ -1544,7 +1542,7 @@ impl<T: ?Sized> Weak<T> {
     /// assert!(!Weak::ptr_eq(&first, &third));
     /// ```
     #[inline]
-    #[unstable(feature = "weak_ptr_eq", issue = "55981")]
+    #[stable(feature = "weak_ptr_eq", issue = "1.37.0")]
```

Comment:
```suggestion
    #[stable(feature = "weak_ptr_eq", since = "1.37.0")]
```

(this is why CI fails)

Reply:
Ah, I didn't see that from the log.

Reply:
@jonas-schievink  Thanks.